person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, parent_permission: true, name: 'Unknown')
+    @id = id
```

Comment:
- [ ] [OPTIONAL] It is good to have a random id, then we dont have to pass `id` anywhere in all classes `Person, Student, or Teacher`.
```ruby
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
.......

....
```